### ISSUE TITLE
Documentation would be nice

### ISSUE THREAD

**Issue by derwaldgeist** [meteor/com.meteor.cordova-update#5]
Hi guys,

I'm building a hybrid Meteor Cordova app and I'm having problems to understand, how exactly the information exchange between Meteor, Cordova and the native side works. Hence, I tried to delve more into the source code, but I am not really sure if I understand it.

As far as I can tell, METEORCordovaURLProtocol seems to establish an internal pseudo-webserver that servers local files stored on the device via the http://meeor.local address. And I'm also assuming that CordovaUpdate establishes this protocol handler.

Yet, this is all just a guess, and it would be highly appreciated if even those "internal plugins" would be documented. At the moment, there is even no comments in the source code, which is not really nice for other developers ;-)

I would be especially interested, what role the method sendPluginResult() plays exactly.

Background of my question: I've noticed that the handleOpenURL JS calls are not being executed if my hybrid app sends this too early, e.g. before Meteor has started up completely. Hence, I'm trying to figure out how I could hook into the event management process to implement a kind of command queue that regularly polls if Meteor is running to send commands safely. Maybe there is already something like that in Cordova (I've seen some queue code there), but I'm not too deep into that code already.

TL;DR: More documentation about the native side of Meteor Cordova would highly be appreciated.
